tests/test_c_parser.py

```diff
@@ -1674,7 +1674,7 @@ def test_unified_string_literals(self):
         self.assertEqual(d1, ['Constant', 'string', '"hello"'])
 
         d2 = self.get_decl_init('char* s = "hello" " world";')
-        self.assertEqual(d2, ['Constant', 'string', '"hello world"'])
+        self.assertEqual(d2, ['Constant', 'string', '"hello"" world"'])
```

Comment:
Unfortunately, this makes no sense in the general case - it goes against language semantics (see the discussion on the issue)

Reply:
Sadly the general case is wrong in particular cases and cannot be extended to cover these